.github/workflows/release.yaml

```diff
@@ -36,21 +21,32 @@ permissions:
 jobs:
 
    publish:
-      if: github.repository == 'Kantis/ks3' && github.event.workflow_run.conclusion == 'success'
+      # only publish when
+      # * on the main repo,
+      # * if triggered by a workflow, the workflow was successful
+      if: >-
+         github.repository == 'Kantis/ks3'
+         &&
+         (!github.event.workflow_run || github.event.workflow_run.conclusion == 'success')
```

Comment:
Do we care to publish if there's failing tests normally? Otherwise we could simplify this with just ` && success()`?

Reply:
there aren't any tests in `release.yml`, but it will get triggered either manually `workflow_dispatch`, or when the build-workflow finishes. But GitHub doesn't provide a good way to check if the triggering workflow was successful, hence this manual check. I think that `success()` only checks the current workflow - not the triggering workflow.

Reply:
https://github.com/orgs/community/discussions/26238

Reply:
Ah, I see :)